drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -137,6 +140,7 @@ function DragAndDropEditBlock(runtime, element, params) {
                         $fbkTab.addClass('hidden');
                         $zoneTab.removeClass('hidden');
                         self.scrollToTop();
+                        $zoneTab.find('input:first').select();
 
                         $(this).one('click', function loadThirdTab(e) {
```

Comment:
@cahrens Thanks for pointing that out. I removed the `preventDefault` calls for the a-tags that are now buttons. (Taking over for @mtyaka since he is away this week.)